lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -1233,18 +1299,42 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
 
   def slot_names(%__MODULE__{is_full_screen: false}), do: [:large]
 
-  def slot_names(%__MODULE__{} = t) do
-    if dual_screen_alert?(t),
-      do: [:full_body],
-      else: [:paged_main_content_left]
+  def slot_names(
+        %__MODULE__{
+          alert: alert,
+          all_platforms_at_informed_station: all_platforms_at_informed_station
+        } = t
+      ) do
+    cond do
+      Alert.is_partial_station_closure?(alert, all_platforms_at_informed_station) ->
+        [:paged_main_content_left]
+
+      dual_screen_alert?(t) ->
+        [:full_body]
+
+      true ->
+        [:paged_main_content_left]
```

Comment:
Would it make more sense to change the criteria for `dual_screen_alert?` to exclude partial closures? That seems like it does describe what we're doing, and then this conditional would remain simple to follow rather than a "maybe A, or maybe B, or maybe A after all" thing.